src/generated/sdk.ts

```diff
@@ -10,11 +10,15 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
+  _Any: any;
```

Comment:
@jekabs-karklins I am not sure why we have/need this `_Any` scalar.

Reply:
99% percent sure it is from Apollo federation, as we use it for generate the schema, even if we don't use the gateway or other features.

Reply:
Thanks good to know! 🚀